crates/fluvio-connector/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "fluvio-connector"
```

Comment:
seems like this should be `fluvio-connector-manager` or since `fluvio-connector` might be reserved for connector framework?